pkg/xds/envoy/access_logs.go

```diff
@@ -23,18 +27,53 @@ func convertLoggingBackends(backends []*v1alpha1.LoggingBackend) ([]*filter_acce
 	return result, nil
 }
 
-func convertLoggingBackend(backend *v1alpha1.LoggingBackend) (*filter_accesslog.AccessLog, error) {
+func convertLoggingBackend(backend *v1alpha1.LoggingBackend, proxy *core_xds.Proxy) (*filter_accesslog.AccessLog, error) {
 	format := AccessLogDefaultFormat
 	if backend.Format != "" {
 		format = backend.Format
 	}
+	services := proxy.Dataplane.Spec.Tags().Values(v1alpha1.ServiceTag)
+	service := "unknown"
+	if len(services) > 0 {
+		service = services[0]
+	}
+	format = strings.ReplaceAll(format, "%KUMA_DOWNSTREAM_CLUSTER%", service)
+
 	if file, ok := backend.GetType().(*v1alpha1.LoggingBackend_File_); ok {
 		return fileAccessLog(format, file)
+	} else if tcp, ok := backend.GetType().(*v1alpha1.LoggingBackend_Tcp_); ok {
+		return tcpAccessLog(format, tcp, proxy.Id)
 	} else {
 		return nil, errors.Errorf("could not convert LoggingBackend of type %T to AccessLog", backend.GetType())
 	}
 }
 
+func tcpAccessLog(format string, tcp *v1alpha1.LoggingBackend_Tcp_, id core_xds.ProxyId) (*filter_accesslog.AccessLog, error) {
+	fileAccessLog := &accesslog.HttpGrpcAccessLogConfig{
+		CommonConfig: &accesslog.CommonGrpcAccessLogConfig{
+			LogName: fmt.Sprintf("%s;%s", tcp.Tcp.Address, format),
+			GrpcService: &core.GrpcService{
+				TargetSpecifier: &core.GrpcService_GoogleGrpc_{
+					GoogleGrpc: &core.GrpcService_GoogleGrpc{
```

Comment:
I think, usage of `GoogleGrpc` is not mandatory in this case (unlike with mTLS).

Although, I don't know what are the downsides to using it